lib/rspec/mocks/error_generator.rb

```diff
@@ -304,6 +304,22 @@ def count_message(count, expectation_count_type=nil)
       def times(count)
         "#{count} time#{count == 1 ? '' : 's'}"
       end
+
+      def formatted_single_message(args)
+        "(" + args.first.map(&:inspect).join(", ") + ")"
+      end
```

Comment:
If the args here are rspec-matchers this will wrongly use `inspect` rather than `description`.  The logic for this is already handled by `format_args` and `args_list` so it'd be best to re-use those and not create yet another method that formats an arg list.